pyMSPannotator/libs/services/__init__.py

```diff
@@ -1,4 +0,0 @@
-from pyMSPannotator.libs.services.CIR import CIR
-from pyMSPannotator.libs.services.CTS import CTS
-from pyMSPannotator.libs.services.NLM import NLM
-from pyMSPannotator.libs.services.PubChem import PubChem
```

Comment:
Doesn't deleting this file cause any issues regarding importing/exporting the package etc.?

Reply:
It is not deleted, just GitHub probably didn't track the change correctly. File is still there, just `pyMSPannotator` is renamed to `MSMetaEnhancer`.